updated email address and author in setup.py files

orbeckst opened the pull request:
Fixes #3174 

Changes made in this Pull Request:
- use mdanalysis@numfocus.org as official address
- use MDAnalysis Development Team as author
- close  #3174


PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [x] Issue raised/referenced?

orbeckst commented on package/setup.py:
```diff
@@ -597,10 +597,10 @@ def long_description(readme):
-          author='Naveen Michaud-Agrawal',
-          author_email='naveen.michaudagrawal@gmail.com',
+          author='MDAnalysis Development Team',
+          author_email='mdanalysis@numfocus.org',
```
Are we happy with _MDAnalysis Development Team_ as author?

lilyminium replied:
Yup, I'm happy with it. I'll let @richardjgowers / @IAlibay / @nmichaud approve the PR though.

orbeckst commented on package/setup.py:
```diff
@@ -597,10 +597,10 @@ def long_description(readme):
-          author_email='naveen.michaudagrawal@gmail.com',
+          author='MDAnalysis Development Team',
+          author_email='mdanalysis@numfocus.org',
           maintainer='Richard Gowers',
```
Is @IAlibay current maintainer or should this still be @richardjgowers ?

IAlibay replied:
@richardjgowers still holds all the keys, I'd like to think of myself as the sith apprentice 🙃 

That being said, if we're looking to change it, shouldn't the maintainers be "The MDAnalysis Core Developer Team"?

orbeckst replied:
I like your suggestion. At least then blame gets properly apportioned.

@richardjgowers what's your opinion?

tylerjereddy replied:
Just for another data point--SciPy mentions the person holding the release manager position at a given time in the online docs, but not in the source as far as I know: https://scipy.github.io/devdocs/dev/governance/people.html?highlight=tyler%20reddy#release-manager

maintainers is "SciPy Developers"

orbeckst replied:
In response to @IAlibay and @richardjgowers comments I am changing the maintainer name to the official title [MDAnalysis Core Developers](https://www.mdanalysis.org/about/#governance). 

(I was striving for grammatical uniformity but If you prefer the article "The" in front I can be swayed.)

orbeckst commented:
@MDAnalysis/coredevs are you happy with this change? Pinging everyone for authorship changes...

I'd appreciate a review from @nmichaud .

IAlibay reviewed (commented):
lgtm, just the one comment

tylerjereddy reviewed (approved):
LGTM -- if Irfan and Richard are still doing some heavy lifting for releases I'd be +1 for their roles to be recognized in the docs somewhere. I can't remember if we have a kind of "roles" section.

We also probably have a fairly clear candidate for the person who leads doc maintenance too, right? I guess maintaining those kinds of lists gets a bit awkward when people shift around a bit, although you can just add an "emeritus" section to be a bit more graceful and say you'll welcome people back with open arms when you do make PRs to change them.

nmichaud commented:
I volunteer to be the first member of the emeritus section :)

Naveen


On Fri, Apr 23, 2021, 7:37 PM Tyler Reddy ***@***.***> wrote:

> ***@***.**** approved this pull request.
>
> LGTM -- if Irfan and Richard are still doing some heavy lifting for
> releases I'd be +1 for their roles to be recognized in the docs somewhere.
> I can't remember if we have a kind of "roles" section.
>
> We also probably have a fairly clear candidate for the person who leads
> doc maintenance too, right? I guess maintaining those kinds of lists gets a
> bit awkward when people shift around a bit, although you can just add an
> "emeritus" section to be a bit more graceful and say you'll welcome people
> back with open arms when you do make PRs to change them.
> ------------------------------
>
> In package/setup.py
> <https://github.com/MDAnalysis/mdanalysis/pull/3239#discussion_r619540800>
> :
>
> > @@ -597,10 +597,10 @@ def long_description(readme):
>                         'trajectories generated by CHARMM, Gromacs, NAMD, LAMMPS, or Amber.'),
>            long_description=LONG_DESCRIPTION,
>            long_description_content_type='text/x-rst',
> -          author='Naveen Michaud-Agrawal',
> -          ***@***.***',
> +          author='MDAnalysis Development Team',
> +          ***@***.***',
>            maintainer='Richard Gowers',
>
> Just for another data point--SciPy mentions the person holding the release
> manager position at a given time in the online docs, but not in the source
> as far as I know:
> https://scipy.github.io/devdocs/dev/governance/people.html?highlight=tyler%20reddy#release-manager
>
> maintainers is "SciPy Developers"
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/MDAnalysis/mdanalysis/pull/3239#pullrequestreview-643719064>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAE6H7TR7QMCNKEC2XYNHLDTKIADPANCNFSM43NX4OBA>
> .
>

orbeckst commented:
ping @nmichaud @richardjgowers for a quick review, please — your names are getting changed so it would be good to have your official approval for the change. Thanks.

richardjgowers reviewed (approved):
Yeah I didn't even realise I was personally listed. I think maintainer as the collective is better

pep8speaks commented:
Hello @orbeckst! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:

* In the file [`package/setup.py`](https://github.com/MDAnalysis/mdanalysis/blob/8ad3932bceb85423da5c7edfd89ec88091936f90/package/setup.py):

> [Line 609:80](https://github.com/MDAnalysis/mdanalysis/blob/8ad3932bceb85423da5c7edfd89ec88091936f90/package/setup.py#L609): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (90 > 79 characters)
> [Line 610:80](https://github.com/MDAnalysis/mdanalysis/blob/8ad3932bceb85423da5c7edfd89ec88091936f90/package/setup.py#L610): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (86 > 79 characters)

* In the file [`testsuite/setup.py`](https://github.com/MDAnalysis/mdanalysis/blob/8ad3932bceb85423da5c7edfd89ec88091936f90/testsuite/setup.py):

> [Line 125:80](https://github.com/MDAnalysis/mdanalysis/blob/8ad3932bceb85423da5c7edfd89ec88091936f90/testsuite/setup.py#L125): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (96 > 79 characters)
> [Line 127:80](https://github.com/MDAnalysis/mdanalysis/blob/8ad3932bceb85423da5c7edfd89ec88091936f90/testsuite/setup.py#L127): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (89 > 79 characters)

orbeckst commented:
I changed the maintainer name. 

I also updated metadata links and added a few new ones to show up in the pypi page
- User Guide
- Blog
- Twitter
- Discord (this one is a bit iffy because you first need the invite but posting the invite link there will invite again people who are already on our discord as opposed to just letting them go to the discord server)

nmichaud reviewed (approved):
Looks good to me.
